app/controllers/bookings_controller.rb

```diff
@@ -1,9 +1,27 @@
 class BookingsController < ApplicationController
+  def index
+    @bookings = Booking.all
+  end
+  
   def new
+    @costume = Costume.find(params[:costume_id])
     @booking = Booking.new
+    @costume.costume_id = @costume
```

Comment:
I think we need to look at the ones that's already there as well. This line might not be needed